pkgs/development/python-modules/duckduckgo-search/default.nix

```diff
@@ -6,35 +6,30 @@
   setuptools,
   click,
   primp,
-
-  # Optional dependencies
   lxml,
 }:
 
 buildPythonPackage rec {
   pname = "duckduckgo-search";
-  version = "8.0.4";
+  version = "8.1.0";
   pyproject = true;
 
-  disabled = pythonOlder "3.8";
+  disabled = pythonOlder "3.9";
```

Comment:
This line can actually be removed too.

Reply:
Why? https://github.com/deedy5/ddgs/blob/3e0e023abdd5313bcdba718609cf4b7585654b2e/pyproject.toml#L9